### PR TITLE
Single source of truth

### PR DESCRIPTION
**Pull request by dataders**
## before merging
- [x] merge https://github.com/dbt-labs/docs.getdbt.com/pull/3055

the rationale for this PR is to consolidate information to a single-source-of-truth for the following reasons:
1. The profile parameter description table is out-of-date
2. with the dbt Cloud integration imminent, the README of this adapter repo is less likely than ever to be the place users go to for documentation

from my perspective, the dbt docs site is the natural candidate for this information to live because it:
1. users would naturally find themselves here
2. is open-source -- anyone can contribute edits via the "edit this page" option on the live site
3. has UX features such as tabbing, snippets
4. is supported by both dbt Labs' docs and partnerships teams
5. virtually all adapter maintainers have accepted this convention so far

## Checklist

- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [x] `README.md` updated and added information about my change
- [ ] I have run `changie new` to [create a changelog entry](https://github.com/starburstdata/dbt-trino/blob/master/CONTRIBUTING.md#Adding-CHANGELOG-Entry)

### CODE REVIEW

**Comment by nghi-ly** [comment]
@dataders : i <3 the idea of having a single source of truth! makes maintenance a lot easier with fewer places to update. future proof ftw!

File: README.md [hunk 1]
```diff
@@ -35,463 +35,17 @@ One frequently asked question in the context of using `dbt` tool is:
...
-models:
-  - name: specific_model
-    config:
-      grants:
-        select: ['reporter', 'bi']
-```
+### Trino- and Starburst-specific configuration
 
-Read everything about grants in the [dbt docs](https://docs.getdbt.com/reference/resource-configs/grants).
+For Trino- and Starburst-specific configuration please refer to "[Starburst (Trino) configurations](https://docs.getdbt.com/reference/resource-configs/trino-configs)" on the dbt docs site
```
Comment (nghi-ly) [inline README.md:48]:
```suggestion
For Trino- and Starburst-specific configuration please refer to "[Starburst (Trino) configurations](https://docs.getdbt.com/reference/resource-configs/trino-configs)" on the dbt docs site.
```

**Comment by dataders** [comment]
@leniartek @przemekak @mdesmet any objections? I give my rationale in the PR body and I've also copied it here below. I spent a great deal of time trying to streamline the relevant information. You can see the final result on the new [Starburst/Trino Setup](https://docs.getdbt.com/reference/warehouse-setups/trino-setup) and [Starburst/Trino Config](https://docs.getdbt.com/reference/resource-configs/trino-configs) pages.

> from my perspective, the dbt docs site is the natural candidate for this information to live because it:
> 
>     1. users would naturally find themselves here
> 
>     2. is open-source -- anyone can contribute edits via the "edit this page" option on the live site
> 
>     3. has UX features such as tabbing, snippets
> 
>     4. is supported by both dbt Labs' docs and partnerships teams
> 
>     5. virtually all adapter maintainers have accepted this convention so far

**Review by hovaesco — APPROVED** [review]
LGTM

**Pull request merged**